.goreleaser.yaml

```diff
@@ -355,3 +367,19 @@ changelog:
 release:
   prerelease: auto
   draft: true
+
+brews:
```

Comment:
I'm not sure this should be in the go-releaser file.  I'm pretty sure that it needs to be added to https://github.com/Homebrew/homebrew-core/blob/main/Formula/o/opentofu.rb instead.